projects/igniteui-angular/src/lib/expansion-panel/expansion-panel-header.component.html

```diff
@@ -0,0 +1,16 @@
+
+<div tabindex="0" role = "button" [attr.aria-disabled] = "disabled" [attr.aria-expanded] = "isExpanded" [attr.aria-controls] = "controls">
+<ng-template #iconContainer>
+    <ng-content *ngIf="iconTemplate" select="igx-expansion-panel-icon"></ng-content>
+    <igx-icon *ngIf="!iconTemplate" fontSet="material" [name]="panel.collapsed? 'expand_more':'expand_less'"></igx-icon>
+</ng-template>
+<ng-container *ngIf="iconPosition === 'left'">
+    <ng-container *ngTemplateOutlet="iconContainer"></ng-container>
+</ng-container>
+<ng-content select="igx-expansion-panel-title"></ng-content>
+<ng-content select="igx-expansion-panel-description"></ng-content>
+<ng-content></ng-content>
+<ng-container *ngIf="iconPosition === 'right'">
```

Comment:
Let's handle the icon positioning using CSS by changing the class modifier on the icon container? For instance if we could have two classes for the icon container - `igx-expansion-panel__header-icon--start` and `igx-expansion-panel__header-icon--end`. I don't see the point of having two conditions that output the same result regardless of the provided input value they check against - in this case the icon position. Also, if we have to still check for the icon position inside the template, wouldn't it be better to use the enumeration members, instead of hard-coded string values to do the checks?